site-src/blog/2023/0801-welcome-gamma.md

```diff
@@ -0,0 +1,125 @@
+---
+description: >
+  We are excited to announce the v0.8.0 release of Gateway API, where the GAMMA
+  project has reached implementable!
+---
+
+# Gateway API: Welcome, GAMMA!
+
+<small style="position:relative; top:-30px;">
+  :octicons-calendar-24: August 01, 2023 ·
+  :octicons-clock-24: 5 min read
+</small>
+
+We are thrilled to announce the v0.8.0 release of the Gateway API! With this
+release, the work the GAMMA initiative has been doing over the past year has
+reached [Experimental status][status], with two conformant implementations at
+present (Linkerd and Istio). We look forward to your feedback!
+
+As [Experimental][status] features, you will find the GAMMA CRDs in the
```

Comment:
nit: is it confusing to call these the "GAMMA CRDs" when it's just essentially the same HTTPRoute? Furthermore, does this mean that someone using HTTPRoute in the standard channel (take a GKE cluster for example) can't use GAMMA routing (e.g. in Istio)?

Reply:
Agreed this is worded a bit awkwardly - because GAMMA hasn't actually introduced any new fields (just codified the behavior of one potential kind, `Service` when used as a value for a Route ParentRef) - I'm pretty sure using the actual experimental channel CRDs would _not_ be necessary.

Reply:
Nice catch – gonna delete that sentence. 🙂